plan/plan.go

```diff
@@ -57,7 +57,11 @@ type Planner interface {
 
 // Row represents a record row.
 type Row struct {
-	Data    []interface{}
+	// Data is the output record data for current Plan.
+	Data []interface{}
+	// FromData is the first origin record data, generated by From.
+	FromData []interface{}
```

Comment:
No, if data is from index plan, it is also treated as "From". 

The from means the origin row came from, it can be table, index, etc.
Btw, if the row is from index plan, it is also from the table, only that we use index to fetch this row.